app/models/grape.rb

```diff
@@ -0,0 +1,16 @@
+class Grape < ActiveRecord::Base
+  def cut
+    "cut cut!"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->